### ISSUE TITLE
Complete Test Suite

### ISSUE THREAD

**Issue by kborchers** [jquery-support/react-globalize#52]
Test all features, options, etc for each component based on the Globalize API

**Comment by kborchers** [comment]
@rxaviers would you mind adding a checklist for each feature we should test for each component based on the Globalize API?

**Comment by necolas** [comment]
What is `scope` for? It's undocumented and untested at the moment

**Comment by rxaviers** [comment]
> What is `scope` for? It's undocumented and untested at the moment

True, it was an experimental feature. You can find some doc at the description of #17. We should either have it properly working and properly documented or drop it.
